art/estimators/generation/tensorflow_gan.py

```diff
@@ -0,0 +1,161 @@
+# MIT License
```

Comment:
Please move this file to a separate module `gan`.

Reply:
addressed